airbyte/metadata.csv

```diff
@@ -0,0 +1,52 @@
+airbyte.metrics_reporter.num_unusually_long_syncs,gauge,,,,Number of unusual long syncs compared to their historic performance.,0,airbyte,num unusually long syncs,
+airbyte.metrics_reporter.oldest_pending_job_age_secs,gauge,,second,,Oldest pending job in seconds.,0,airbyte,oldest pending job age secs,
+airbyte.metrics_reporter.oldest_running_job_age_secs,gauge,,second,,Oldest running job in seconds.,0,airbyte,oldest running job age secs,
+airbyte.metrics_reporter.overall_job_runtime_in_last_hour_by_terminal_state_secs,distribution,,second,,Overall job runtime - scheduling and execution for all attempts - for jobs that reach terminal states in the last hour. Tagged by terminal states.,0,airbyte,overall job runtime in last hour by terminal state secs,
+airbyte.orchestrator.source_hearbeat_failure,count,,,,Fail a replication because the source missed an heartbeat.,0,airbyte,source hearbeat failure,
+airbyte.server.breaking_change_detected,count,,,,A breaking schema change has been detected.,0,airbyte,breaking change detected,
+airbyte.server.schema_change_auto_propagated,count,,,,A schema change have been propagated.,0,airbyte,schema change auto propagated,
+airbyte.worker.activity.check_connection,count,,connection,,Increments when we start a check connection activity.,0,airbyte,activity check connection,
+airbyte.worker.activity.dbt_transformation,count,,,,Increments when we start a dbt transformation activity.,0,airbyte,activity dbt transformation,
+airbyte.worker.activity.discover_catalog,count,,,,Increments when we start a discover catalog activity.,0,airbyte,activity discover catalog,
+airbyte.worker.activity.failure,count,,,,Generic metric for incrementing when an activity fails. Add activity name to attributes.,0,airbyte,activity failure,
+airbyte.worker.activity.normalization,count,,,,Increments when we start a normalization activity.,0,airbyte,activity normalization,
+airbyte.worker.activity.normalization_summary_check,count,,,,Increments when we start a normalization summary check activity.,0,airbyte,activity normalization summary check,
+airbyte.worker.activity.refresh_schema,count,,,,Increments when we start a refresh schema activity.,0,airbyte,activity refresh schema,
+airbyte.worker.activity.replication,count,,,,Increments when we start a replication activity.,0,airbyte,activity replication,
+airbyte.worker.activity.spec,count,,,,Increments when we start a spec activity.,0,airbyte,activity spec,
+airbyte.worker.activity.submit_check_destination_connection,count,,connection,,Increments when we start a submit check connection activity.,0,airbyte,activity submit check destination connection,
+airbyte.worker.activity.submit_check_source_connection,count,,connection,,Increments when we start a submit check connection activity.,0,airbyte,activity submit check source connection,
+airbyte.worker.activity.webhook_operation,count,,,,Increments when we start a webhook operation activity.,0,airbyte,activity webhook operation,
```

Comment:
```suggestion
airbyte.worker.activity.webhook_operation,count,,operation,,Increments when we start a webhook operation activity.,0,airbyte,activity webhook operation,
```
maybe?

Reply:
This seems to be rather about an "airbyte activity" whatever that is.